<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualTerminalWidget</class>
 <widget class="QWidget" name="VirtualTerminalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>94</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VirtualTerminalWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
  border-width: 0px
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="videoShowedLabel">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="cutFileAction">
   <property name="text">
    <string>录制分段</string>
   </property>
   <property name="toolTip">
    <string>录制分段</string>
   </property>
  </action>
  <action name="stopReceiveVideoAction">
   <property name="text">
    <string>停止接收视频</string>
   </property>
   <property name="toolTip">
    <string>停止接收视频</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CoolView.qrc"/>
 </resources>
 <connections/>
</ui>
